src/client/theme-default/Layout.vue

```diff
@@ -34,10 +34,24 @@
     <main v-else>
       <Page>
         <template #top>
-          <slot name="page-top" />
+          <slot name="page-top">
```

Comment:
Maybe we should rename this slot to `page-top-ads` or something? Since people might want to add something other than ads in this section, while preserving the ads 😳 

Same goes for `page-bottom` as well (page-bottom-ads).

Reply:
Sure! Let me see if we can have both slots without one being nested into each other, it should be good

Reply:
Pushed the changes, let me know what you think. The preview should also work to make sure it works. I tested on Vuex repo

Reply:
Niiice! Looking good I think 🙌